### DISCUSSION TITLE
ideas after tests

### DISCUSSION THREAD

**Discussion by bfd69** [linagora/linshare#309] [Ideas]
Hello there,
First let me congratulate for the amazing job, linshare is just what i was missing. I didnt tests it fully, only the online demo, so i have a bunch of idea that i would like to humbly share with you, perhaps some  are just a misunderstanding of linshare on my behalf.
so : 
- is it possible to organise the documents that one would like to share with folders or a folder tree ? at leat some tags ? the latter would come with a tool to mass search and tag..
- is it possible to share those folders ?
- i didnt find a way to filter only files that are shared.
- i didnt find a way to see the users whith whom i shared
- when i create a deposit invitation it cannot start immediatly
- when i share files, i cannot set start time, only date : (i'm planning to use it for exams too)

what i like in linshare : 
openldap support
openid support
clean and reactive interface
full online demo

Bravo !
Best regards

**Comment by dieptran88** [comment]
Hello, some points to answer your questions:



> * is it possible to organise the documents that one would like to share with folders or a folder tree ? at leat some tags ? the latter would come with a tool to mass search and tag..

**=>> the folder feature in Myspace (where you can share files) is in our roadmap, hopefully it will be release soon.** 
> * is it possible to share those folders ?

**=> When it is done, of course you can share folders from Myspace.** 
> * i didnt find a way to filter only files that are shared.

**=>> Thanks for good ideal, we will consider it. Currently you can sort files which is shared or not.**
> * i didnt find a way to see the users whith whom i shared

**=>> You can find recipient when you select View detail of file, then select tab Shares, and click on each recipient to see more details:**
![image](https://github.com/linagora/linshare/assets/68209176/36f2257b-1d9a-4f35-a4c2-b6395d3136d9)

> * when i create a deposit invitation it cannot start immediatly

**=>> It will be fixed soon**
> * when i share files, i cannot set start time, only date : (i'm planning to use it for exams too)

**=>>We will consider this** 
>
